t/op/ref.t

```diff
@@ -290,7 +292,10 @@ sub mymethod {
 $string = "bad";
 $object = "foo";
 $string = "good";
-$main'anonhash2 = "foo";
+{
+    no warnings "syntax", "deprecated";
```

Comment:
Ditto `qw(...)` here and further below

Reply:
Fixed in a few places here, and in some other tests.